### ISSUE TITLE
SDF texture generation

### ISSUE THREAD

**Issue by Dimev** [mooman219/fontdue#126]
Would it make sense to also add SDF font generation to fontdue?

This would allow easier zooming/rotation of text when paired with the right shader, and maybe also reduce the size of the glyph texture atlas.

**Comment by mooman219** [comment]
Yes that would make sense, I've been playing around with that idea for a bit, but it's a bit slow to generate.

**Comment by therocode** [comment]
I second this. Currently hitting a roadblock where text cannot be rotated without looking blocky/bad due to the pixel grid effectively being rotated against the display buffer.

**Comment by grimtin10** [comment]
i third this
i like fontdue for how lightweight and fast it is (and i don't particularly feel like pulling in a different crate to handle SDF font generation)
i may look into how SDF font generation works and make a PR at some point but it won't be soon
